snapshots/happiness/2024-06-09/happiness.xls.dvc

```diff
@@ -0,0 +1,31 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: World Happiness Report
+    description: |-
+      The World Happiness Report is a partnership of Gallup, the Oxford Wellbeing Research Centre, the UN Sustainable Development Solutions Network, and the WHR’s Editorial Board.
+      It reviews the state of happiness in the world today and shows how the science of happiness explains personal and national variations in happiness.
+    date_published: "2024-03-08"
+    version_producer: 2024
+
+    # Citation
+    producer: "University of Oxford: Wellbeing Research Centre"
```

Comment:
This could be simply "Wellbeing Research Centre". That's how they call themselves in multiple public places. And you mention Oxford in the full citation, so that should be fine.